WordPress/gutenberg
# Workflows: Sync assets to plugin repo upon change in trunk

Fixes https://github.com/WordPress/gutenberg/issues/67835.

## What?
Sync the `assets/` directory to the plugin svn repo whenever a file in that directory is changed as part of a commit to `trunk`

## Why?
To sync the plugin repo's assets for the Gutenberg plugin with the latest changes from git.

## How?
By checking out the `assets/` directory from git, and using it to replace the contents of the assets in the Gutenberg plugin's svn directory.

## Testing Instructions
I have a test SVN repo that I had already set up to include the `assets/` directory from the GB plugin repo, plus a `test.txt` file in that directory.

I tested this workflow by adding a commit to set the plugin repo URL to that test svn repo, and force-pushing the git branch to my personal Gutenberg fork's `trunk`. The workflow run passed, you can see it [here](https://github.com/ockham/gutenberg/actions/runs/12394491777/job/34598075283). (It removed the `test.txt` file, as it's not present in git).

The commit message recorded in my test SVN repo is `Sync assets for commit 97de59b764c9017242d1da44a034015fc262fb80`.
